mypy/messages.py

```diff
@@ -241,6 +241,12 @@ def format_simple(self, typ: Type, verbosity: int = 0) -> str:
           None -> None
           callable type -> "" (empty string)
         """
+        ret = self._format_simple(typ, verbosity)
+        if ret == "Module":
```

Comment:
Only one this one line in the source code. (In the output I am fine with the double quotes.) But the line of code that generates the special value `'Modules'` also uses single quotes, as does most of the surrounding code, so I'd like this line to be consistent.